packages/icons/stories/Icon.tsx

```diff
@@ -240,64 +258,73 @@ export const IconItem = ({
 	return (
 		<div {...rest}>
 			{isFound ? (
-				<div role="button" onClick={onClickHandler} onKeyPress={onClickHandler} tabIndex={0}>
+				<div
+					style={{
+						display: 'flex',
+						gap: tokens.coralSpacingM,
+						paddingBlock: tokens.coralSpacingXs,
+					}}
+				>
 					<div
 						style={{
 							display: 'flex',
-							gap: tokens.coralSpacingM,
-							paddingBlock: tokens.coralSpacingXs,
+							justifyContent: 'center',
+							alignItems: 'center',
+							width: tokens.coralSizingM,
+							height: tokens.coralSizingM,
+							border: `${tokens.coralBorderSSolid} ${tokens.coralColorNeutralBorderWeak}`,
+							boxShadow: tokens.coralElevationShadowAccent,
+							borderRadius: tokens.coralRadiusS,
 						}}
 					>
 						<div
-							style={{
-								display: 'flex',
-								justifyContent: 'center',
-								alignItems: 'center',
-								width: tokens.coralSizingM,
-								height: tokens.coralSizingM,
-								border: `${tokens.coralBorderSSolid} ${tokens.coralColorNeutralBorderWeak}`,
-								boxShadow: tokens.coralElevationShadowAccent,
-								borderRadius: tokens.coralRadiusS,
-							}}
+							role="button"
+							onClick={onIconClickHandler}
+							onKeyPress={onIconClickHandler}
```

Comment:
I am a bit worried about this onKeyPress.
At least could you check the event to only donwload if enter key or click ?

Reply:
Right, i can alter this ! 👍🏼